docs/content/release_notes/changelog_1.9.0.md

```diff
@@ -2,4 +2,6 @@
 
 IBM® block storage CSI driver 1.9.0 adds new support and enhancements:
 - New stretched topology support for SAN Volume Controller storage systems
+- New Call Home support for Spectrum Virtualize family storage systems
+- New RHEL 8.x support for x86 architectures
```

Comment:
should RHEL 8.x be part of the call home diff?

Reply:
i'm not sure why it's showing up as new. it's also in the main branch.

Reply:
![image](https://user-images.githubusercontent.com/57443811/157642062-66d3d417-70c9-4c93-bd85-48d705bc59ab.png)
try to merge the main branch to this branch